### PR TITLE
Fixing invalid name for metrics component

### PR DESCRIPTION
**Pull request by matzew**
... and updating metrics settings on the deployment yaml

### CODE REVIEW

File: config/operator.yaml [hunk 1]
```diff
@@ -25,12 +25,17 @@ spec:
...
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: CONFIG_LOGGING_NAME
+              value: config-logging
+            - name: CONFIG_OBSERVABILITY_NAME
+              value: config-observability
             - name: METRICS_DOMAIN
-              value: knative.dev/operator-operator
+              value: knative.dev/eventing-operator
```
Comment (matzew) [inline config/operator.yaml]: changed to `eventing-operator` sicne that makes more sense ...

File: config/operator.yaml [hunk 2]
```diff
@@ -25,12 +25,17 @@ spec:
                 fieldRef:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
+              value: "knative-serving-operator"
```
Comment (matzew) [inline config/operator.yaml]: aligned the `OPERATOR_NAME` with how it is done for `serving-operator` ....

Comment (houshengbo) [reply config/operator.yaml]: Shouldn't it be knative-eventing-operator?

Comment (matzew) [reply config/operator.yaml]: @houshengbo thanks - I just corrected that !

**Pull request merged**